ginga/rv/plugins/LineProfile.py

```diff
@@ -275,45 +303,55 @@ def _slice(self, naxes, mk):
 
         # For axis > 3
         for i in range(2, naxes):
-            slice_obj[i] = int(round(self.image.revnaxis[i-2] + 1))
+            if i != i_sel:
+                slice_obj[i] = 0  # int(round(self.image.revnaxis[i-2] + 1))
```

Comment:
I think this is so that the plugin updates correctly when then the slice is changed through `MultiDim` or other method that manipulates an `AstroImage`.

The `naxispath` is the path through the multidimensional data *after* the first two axes.  And `revnaxis` is that reversed.  The loop is iterating through the dimensions beyond 2.